src/test/components/FlameGraph.test.js

```diff
@@ -5,66 +5,114 @@
 // @flow
 import * as React from 'react';
 import FlameGraph from '../../components/flame-graph';
-import { render } from 'react-testing-library';
+import { render, fireEvent } from 'react-testing-library';
 import { Provider } from 'react-redux';
 import mockCanvasContext from '../fixtures/mocks/canvas-context';
 import { storeWithProfile } from '../fixtures/stores';
 import { getBoundingBox } from '../fixtures/utils';
 import { getProfileFromTextSamples } from '../fixtures/profiles/processed-profile';
-import { changeInvertCallstack } from '../../actions/profile-view';
+import {
+  changeInvertCallstack,
+  changeSelectedCallNode,
+} from '../../actions/profile-view';
+import { selectedThreadSelectors } from '../../selectors/per-thread';
 import mockRaf from '../fixtures/mocks/request-animation-frame';
+import { ensureExists } from '../../utils/flow';
 
-it('renders FlameGraph correctly', () => {
-  const flushRafCalls = mockRaf();
-  window.devicePixelRatio = 1;
```

Comment:
I don't see this `window.devicePixelRatio` anywhere in the new code, is that expected?

Reply:
I removed it because it wasn't used (or so I believed) and the test ran fine without it.

Reply:
It's used in https://github.com/firefox-devtools/profiler/blob/423cfb5dc51809b0d1eabcc5cbab30997cc9d532/src/components/shared/chart/Canvas.js#L97-L99

But I see it's been added in jsdom 11.6 (we upgraded to 13 recently), so that's the reason ir works!

Reply:
Thanks for explaining why it worked :-)